dtmi/aaeon/gene_whu6-1.json

```diff
@@ -0,0 +1,21 @@
+{
+    "@id": "dtmi:AAEON:GENE_WHU6;1",
+    "@type": "Interface",
+    "displayName": "GENE-WHU6",
+    "@context": [
+        "dtmi:iotcentral:context;2",
```

Comment:
The central context is not needed since you are not using anything that require it. Consider to remove it.